### ISSUE TITLE
Create a Python library for using rust-code-analysis

### ISSUE THREAD

**Issue by marco-c** [mozilla/rust-code-analysis#54]
It would be easier for projects using rust-code-analysis to handle their dependency on rust-code-analysis.

**Comment by luigig44** [comment]
Hi! I've done just that: https://github.com/luigig44/rust-code-analysis-python ([rust-code-analysis-python in PyPI](https://pypi.org/project/rust-code-analysis-python/))


It's pretty barebones at the moment, but should work as a pretty much drop-in replacement for anyone using the REST API metrics or comment removal endpoints from Python.

I needed it for another project so it will probably never include much more than those two functionalities, but if someone requests something else in an issue I will look into adding it.

**Comment by marco-c** [comment]
Nice @luigig44!
If you are up for it, we could definitely accept a PR to add this upstream.

**Comment by marco-c** [comment]
@Luni-4 any thoughts?

**Comment by Luni-4** [comment]
Thanks a lot @luigig44 for this project! Out of curiosity, since I'm not a `Python` expert, have you tried https://github.com/astral-sh/uv as package manager?

**Comment by luigig44** [comment]
> have you tried https://github.com/astral-sh/uv as package manager?  

I have not. I do not think it would make any difference because what is failing is Cargo due to missing files in the sdist. You can see the exact error in the issue I linked.

But I may end up trying it anyway.

**Comment by Luni-4** [comment]
Yep, I have seen the issue, just wondering if that problem might be fixed in another way. Ok, let me know if `uv` works better for your use-case.
